components/home/Home.js

```diff
@@ -107,20 +107,20 @@ class Home extends Component {
         );
 
         const SaveIcon = (style) => (
-            <Icon {...style} name='save-outline' />
+            <Icon {...style} name='log-out' />
          );
 
         // right control buttons on navigation bar that changes depending on edit mode
-        const rightControls = () => {
-            const { edit } = this.state
+        const rightControls = (edit) => {
+//            const { edit } = this.state
             if (edit) {
                 return (
                     <View style={styles.rightControlsContainer}>
                         <Layout style={styles.rightControls}>
                             <TopNavigationAction icon={SaveIcon} onPress = {() => {
                                 this.setState({edit: false})
                             }}/>
-                            <Text style={styles.rightControlsText}>Save</Text>
+                            <Text style={styles.rightControlsText}>Finish Edit</Text>
```

Comment:
what happened with "exit editing mode"?

Reply:
That sounds like a great option. I would definitely change it to "exit editing mode" if that was an option appearing in our doc.

Reply:
I think it was listed in our Trello card (https://trello.com/c/kKdHKelu)?

Reply:
I think I remember that. It just looks way too long